.github/workflows/test.yml

```diff
@@ -0,0 +1,68 @@
+name: 🧪 Test
+
+on: 
+  push:
+    branches:
+      - main
+      - dev
+  pull_request_target:
```

Comment:
I am curious how this works? Shouldn't there be some value there?

Reply:
As of my knowlegde, If no value is given to `pull_request_target:`, by default the workflow will be triggered by all events(open, sync, reopen) related to pull requests targeting the base repository.
Although we can give types and  branches filter but we won't be needing those.